apps/bfd-server/dev/api-changelog.md

```diff
@@ -2,9 +2,31 @@
 
 ## BFD-1424 Fix mtus code
 
-Update Carrier claim mapping for mtus code instead of mtus count:
-The new coding:
+Update Carrier claim extension mapping for mtus code instead of mtus count:
+The old coding:
```

Comment:
Maybe we can indicate that we are adding a new extension with the correct url/system for MTUS Count and keeping the old extension with the MTUS Count value that had the incorrect url/system of MTUS Code.  Something to that effect here?  Does that make sense?

Reply:
Oops except I made one mistake and what you added looks and sounds good minus the mistake I made.  Let's change to: "Added a new extension with the correct url/system for MTUS Code and kept the old extension with the MTUS Code value that had the incorrect url/system of MTUS Count"